### ISSUE TITLE
Getting Userinfo, invalid birthadate mapping

### ISSUE THREAD

**Issue by mslawin** [paypal/PayPal-REST-API-issues#128]
### General information

* Environment: Sandbox
* `PayPal-Debug-ID` values: 69c51ab8-eafd-4ad2-8595-87f8f0c562bf
* Language, language version, and OS: Java 1.8.0_131 on macOS 10.12.5 (16F73)

### Issue description

I'm using PayPal-Java-SDK for Login with PayPal integration. When I make request to 'https://api.sandbox.paypal.com/v1/identity/openidconnect/userinfo?schema=openid' using `UserInfo.getUserInfo()`
I get a JSON response with user details and there is 
`"birthday":"1982-08-02"`
Then, this response is deserialized using GSON to object UserInfo and Userinfo object has filed "**birthdate**", not "**birthday**", so the date of birth is not deserialized and is always empty.

**Comment by xiaoleih41** [comment]
I  am looking into this.

**Comment by pp-randy** [comment]
`birthday` is being returned by the API.  We will need to correct the documentation and SDK.

**Comment by pp-randy** [comment]
https://github.com/paypal/PayPal-Java-SDK/blob/master/rest-api-sdk/src/main/java/com/paypal/api/openidconnect/Userinfo.java#L67-L70

**Comment by mslawin** [comment]
Thanks.
I'm not sure if I understand you correctly. Does this mean that you are going to change the field's name to be 'birthday' in Userinfo class?

**Comment by pp-randy** [comment]
Yes, that is the current direction I am going.   Usually the actual API response fields don't get altered.  You can change code now to get up and running with the SDK if you need to, and I would be happy to review a pull request if you change the field.

**Comment by mslawin** [comment]
Hi @pp-randy 
Thanks, I did the change and created a pull request. Please let me know if you have any comments.

Thanks,
Maciej

**Comment by xiaoleih41** [comment]
We will review and get back to you through the pull request.
